RoddyCore/src/de/dkfz/roddy/plugins/LibrariesFactory.groovy

```diff
@@ -152,13 +153,9 @@ public class LibrariesFactory extends Initializable {
             Class foundCoreClass = null;
             for (Package p in getRoddyPackages()) {
                 String className = "${p.name}.${name}"
-                try {
-                    Class _test = groovyClassLoader.loadClass(className)
-                    if (_test)
-                        foundCoreClass = _test
-                } catch (ClassNotFoundException ex) {
-                    // Just catch and ignore, we will fall back to the plugin strategy afterwards! Or search in the next package
-                }
+                foundCoreClass = tryLoadClass(className)
+                if (foundCoreClass) break
```

Comment:
This code is not equivalent to the previous version. Before you didn't break but continued with the for loop.

P.S.: Maybe to make the code clearer, use a while loop. The for-break construct is awkward.